tests/classes/Utils.js

```diff
@@ -135,7 +135,7 @@ describe('Utils', () => {
 
       expect(() => {
         serverless.utils.readFileSync(tmpFilePath);
-      }).to.throw(new RegExp(`in "${tmpFilePath}"`));
+      }).to.throw(new RegExp('YAMLException:'));
```

Comment:
I'm sure I can work something better out. What's the teams philosophy towards modules? Are you trying to minimize deps or anything like that? I can look for a cross platform path-compare lib or just do some basic string substitution of slashes.